src/kernel.ts

```diff
@@ -305,7 +330,6 @@ class Kernel {
   start(id: IKernelId) : void {
```

Comment:
This is more about connecting to a kernel than starting a kernel, and connect goes along with disconnect and reconnect.  Can you rename this method `connect`?